example/testapp/models.py

```diff
@@ -9,10 +9,13 @@ class Car(models.Model):
     def __unicode__(self):
         return self.plate
 
+class m2mprint:
+    def __unicode__(self):
+        return unicode(self.car) + " in " + unicode(self.parkingarea)
 
 class ParkingArea(models.Model):
     name = models.CharField(max_length=50)
-    cars = SortedManyToManyField(Car)
+    cars = SortedManyToManyField(Car, base_class=m2mprint)
 
```

Comment:
follow [pep8](https://www.python.org/dev/peps/pep-0008/) conventions and add an additional blank line